controlnext/models.py

```diff
@@ -82,7 +80,8 @@ class Basin(geomodels.Model):
                             help_text=_("must be unique for grower"))
     location = geomodels.PointField(blank=True, null=True, srid=SRID,
                                     verbose_name=_("location"))
-
+    random_url_slug = models.CharField(max_length=20, unique=True,
+                                       default=random_slug)
```

Comment:
Bij meerdere argumenten zet ik ze liever allemaal onder elkaar. Zeker bij models waar je nogal eens iets toe wilt voegen zoals `verbose_name` en `help_text`. Ik zou in ieder geval `verbose_name` nog even toevoegen.

Qua uitlijning:

    random_url_slug = models.CharField(
        max_length=20, 
        unique=True,
        default=random_slug)